specification/networkanalytics/NetworkAnalytics.Management/examples/2023-11-15/DataProducts_Delete_MaximumSet_Gen.json

```diff
@@ -0,0 +1,18 @@
+{
+  "title": "DataProducts_Delete_MaximumSet_Gen",
+  "operationId": "DataProducts_Delete",
+  "parameters": {
+    "api-version": "2023-11-15",
+    "subscriptionId": "00000000-0000-0000-0000-00000000000",
+    "resourceGroupName": "aoiresourceGroupName",
+    "dataProductName": "dataproduct01"
+  },
+  "responses": {
+    "202": {
```

Comment:
>202 [](http://example.com/codeflow?start=5&length=3)

delete examples often also usually include an empty 200 response if you follow ARM guidelines for delete responses

```
    "200": {}
```

Reply:
In this case if you only have long-running delete, I guess I can see an argument for it always completing with 204.

Reply:
200 response is a vialoation for LRO for delete.
Hence the current response code stays valid.